app/components/Poll/index.js

```diff
@@ -130,8 +138,12 @@ class Poll extends Component<Props, State> {
             <p>{description}</p>
           </div>
         )}
-        {hasAnswered && resultsHidden && <p>Resultatet er skjult</p>}
-        {hasAnswered && !resultsHidden && (
+        {hasAnswered && resultsHidden && !allowedToViewHiddenResults && (
```

Comment:
I think `resultsHidden` and `allowedToViewHiddenResults` can benefit from being merged into one variable. The code is getting a bit hard to read now :upside_down_face:

Reply:
I totally agree that it makes it more complicated, but I was so sure there was a reason it was split into two. I haven't had the chance to test it out yet, but I will very soon.